File: Selnium script for testing/Script with Python/test_scenario_57.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_57():
    """
    Dummy test run for scenario 57
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_57")

    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_851")
    time.sleep(2)
    driver.find_element(By.ID, "user_name").send_keys("test_data_682")
    driver.find_element(By.ID, "user_name").send_keys("test_data_403")
    driver.find_element(By.LINK_TEXT, "Log Out").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_57()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario57.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario57 {
    public static void executeTest57() {
        /*
         * Dummy test run for scenario 57
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_57");

        try {
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.id("user_name")).click();
            driver.findElement(By.id("user_name")).sendKeys("test_data_920");
            Thread.sleep(2000);
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_705");
            Thread.sleep(2000);
            driver.findElement(By.linkText("Log Out")).click();
            driver.findElement(By.className("nav-item")).click();
            Thread.sleep(2000);
            driver.findElement(By.className("nav-item")).click();
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_570");
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest57();
    }
}
